### ISSUE TITLE
Code splitting : Identifier 'React' has already been declared

### ISSUE THREAD

**Issue by camille-hdl** [rollup/rollup#2715]
Hello

- Rollup Version: 1.2.2
- Operating System (or Browser): Chrome 72 on MacOS 10.14.11
- Node Version: 11.6.0

### How Do We Reproduce?

https://github.com/camille-hdl/rollup-starter-code-splitting

(instructions in the readme)  
* Dynamically import a component (that imports `React`) from another file that also imports `React`,
* Build the application, run it in the browser.

### Expected Behavior

`React` is not aliased twice, the page works.

### Actual Behavior

In the output, `React` is aliased twice in a import statement:
```
import { ..., d as React, e as require$$1, f as require$$2, g as React } from './chunk....js';
```
resulting in the following error message in the console:
```
Uncaught (in promise) SyntaxError: Identifier 'React' has already been declared
```

**Comment by camille-hdl** [comment]
update : it seems the issue is not present in older versions of rollup (it worked with 1.0.1)

**Comment by lukastaegert** [comment]
Fix at #2719

**Comment by camille-hdl** [comment]
that was fast ! thank you 👍
